lang_GENERIC_base/Lang.ml

```diff
@@ -160,6 +164,7 @@ let ext_of_lang = function
   | Kotlin -> ["kt"]
   | Lua -> ["lua"]
   | Rust -> ["rs"]
+  | R -> ["r", "R"]
```

Comment:
you didn't compile pfff, because there's a compilation error here. Lists in OCaml are separated by ; not ,.
You use , for tuples.
I'll fix it.

Reply:
Thanks @aryx I'll make sure to build locally next time. I'll have to see what's going on with my editor extension to see why it didn't catch this.